python/ray/data/_internal/progress_bar.py

```diff
@@ -41,7 +41,12 @@ class ProgressBar:
     """Thin wrapper around tqdm to handle soft imports."""
 
     def __init__(
-        self, name: str, total: int, position: int = 0, enabled: Optional[bool] = None
+        self,
+        name: str,
+        total: int,
+        unit: str,
```

Comment:
Nit: Should this be a keyword-argument with a default of `"it"`?

Reply:
That's what I had initially, but @scottjlee suggested [above](https://github.com/ray-project/ray/pull/46432#discussion_r1671345406) to make it required in order to improve overall usability of Ray Data

Reply:
Ah, okay. Sg